docs/tutorials/installation.rst

```diff
@@ -0,0 +1,23 @@
+.. _installation:
+
+**********************
+Installing sunkit-spex
+**********************
+
+Installing the development version
```

Comment:
You should reference the sunpy dev install guide here: https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html#setting-up-a-development-environment although I appreciate that document is aimed at someone who's going to contribute. (I would argue if sunkit-spex is at the point where people are using it but not contributing you should just cut a release)